docs/contributing/contributing-to-nextJS-branch.md

```diff
@@ -0,0 +1,164 @@
+
+## 👨‍💻 Prerequisite Skills to Contribute
+
+### Contribute in public/Profile
+
+- [Git](https://git-scm.com/) 
+- [JSON](https://www.json.org/json-en.html)
+
+### Contribute in Documents
+
+- [Markdown](https://www.markdownguide.org/basic-syntax/)
+
+### Contribute in Components/CSS
+
+- [React](https://reactjs.org/)
+- [Prime React](https://www.primefaces.org/primereact/)
+
+
+---
+
+## 💥 How to Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/EddieHubCommunity/LinkFree/pulls)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/EddieHubCommunity/)
+
+- Take a look at the existing [Issues](https://github.com/EddieHubCommunity/LinkFree/issues) or [create a new issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose)!
+- [Fork the Repo](https://github.com/EddieHubCommunity/LinkFree/fork). Then, create a branch for any issue that you are working on. Finally, commit your work.
+- Create a **[Pull Request](https://github.com/EddieHubCommunity/LinkFree/compare)** (_PR_), which will be promptly reviewed and given suggestions for improvements by the community.
+- Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes proposed in your PR.
+
+---
+
+## ⭐ HOW TO MAKE A PULL REQUEST:
+
+**1.** Start by making a Fork of the [**LinkFree**](https://github.com/EddieHubCommunity/LinkFree) repository. Click on the <a href="https://github.com/EddieHubCommunity/LinkFree/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a>Fork symbol at the top right corner.
+
+Dont forget to uncheck the 'Copy the main branch only' option when forking the repo.
+
+![Screenshot 2022-10-05 235606](https://user-images.githubusercontent.com/32553276/194235257-5b53c801-514f-48d9-ba22-31352c24373c.png)
+
+**2.** Clone your new fork of the repository in the terminal/CLI on your computer with the following command:
+
+```bash
+git clone https://github.com/<your-github-username>/LinkFree
+```
+
+**3.** Navigate to the newly created LinkFree project directory:
+
+```bash
+cd LinkFree
+```
+
+**4.** Set upstream command:
+
+```bash
+git remote add upstream https://github.com/EddieHubCommunity/LinkFree.git
+```
+
```

Comment:
I think the person needs to checkout the next-js branch first and then make their branch from that branch instead of main